asylo/third_party/fortanix/asylo/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "asylo"
+version = "0.1.0"
+authors = ["Mohsen Zohrevandi <mohsen.zohrevandi@fortanix.com>"]
+edition = "2018"
```

Comment:
Please fill out more details here. See https://doc.rust-lang.org/cargo/reference/manifest.html and e.g. https://github.com/fortanix/rust-sgx/blob/master/enclave-runner/Cargo.toml